src/app.ts

```diff
@@ -129,9 +128,10 @@ export const init = async (): Promise<void> => {
   // await Promise.all([initDatabase(), initSlack(), initDiscord()])
 
   const promises = [];
-  promises.push(initDatabase());
+  // promises.push(initDatabase());
```

Comment:
If code is commented out let's just remove it, there's no need to keep it around

```suggestion
```